funny.txt

```diff
@@ -0,0 +1,2 @@
+I don’t trust stairs. They’re always up to something.
+
```

Comment:
Please remove the unnecessary newline. Our linters would complain.

Reply:
Fixed!